components/movementsensor/adxl345/interrupts.go

```diff
@@ -0,0 +1,80 @@
+package adxl345
+
+// addresses relevant to interrupts.
+const (
+	// contains an unsigned time value representing the maximum time that an event must be above the
+	// THRESH_TAP threshold to qualify as a tap event [625 µs/LSB].
+	DurAddr byte = 0x21
+	// contains info on which interrupts have been enabled.
+	IntEnableAddr byte = 0x2E
+	// contains info on which interrupt pin to send each interrupt.
+	IntMapAddr byte = 0x2F
+	// contains info on which interrupt has gone off since the last time this address has been read from.
+	IntSourceAddr byte = 0x30
+	// contains an unsigned time value representing the wait time from the detection of a tap event to the
+	// start of the time window [1.25 ms/LSB].
+	LatentAddr byte = 0x22
+	// contains info on which axes have been turned on for taps (X, Y, Z are bits 2, 1, 0 respectivel).
+	TapAxesAddr byte = 0x2A
+	// contains an unsigned threshold value for tap interrupts [62.5 mg/LSB ].
```

Comment:
All of these comments start with the word "contains". Could those be removed? "An unsigned value representing the maximum duration...", "a bitmask describing...", etc.